.changeset/rotten-camels-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@adyen/adyen-web': patch
+---
+
+Added optional chaining operator to postMessage handler function
```

Comment:
This ends up in the release notes. Maybe we can be more clear about which problem this solves?

Reply:
In my opinion, not really. I think it is better to describe the bug that is fixed rather than the fix itself or point it somewhere else. This also helps the docs team to write better docs and avoid back and forth with questions, and  also improves our release notes by making clear what problem got fixed.

More explanatory sentence for example:
`Fixed issue where 'get-process-message.handler.js' caused exceptions due to stray postMessages`